### DISCUSSION TITLE
Numbering of verses in psalms

### DISCUSSION THREAD

**Discussion by efati** [DivinumOfficium/divinum-officium#4413] [General]
I would like to pick up the problem of verse numbering in psalms. I noticed that it was discussed briefly in #271, but I am not convinced that it is such a big problem.

I have been working on a bilingual printed Psalter for some time, and I decided to use the clementine vulgate source, which is included in this project, mainly because it is organized in the same manner as in breviaries. Unfortunately, the numbering is "_artificial and not that of any edition of scripture_". I would love to update it, to make it (almost) consistent with contemporary translations of the Book of Psalms. I have looked through several of them (new and old ones), and noticed that they are almost consistent in numbering of verses. Take as an example http://vulgate.org or http://vulsearch.sourceforge.net/html/Ps.html.

My idea is the following:

1. Stick with the Tradition wherever it is possible, which means: do not change the divisions into verses.
2. Change the numbering according to http://vulgate.org/ (and other sources).
3. Wherever the numbering changes inside the (current) verses, decide arbitrary to make it reasonable.
4. Omit the first, introductory verses of newer translations (no change).

You can take a look at example changes I made in my branch for Psalms 1-10 (only numbering):
https://github.com/efati/divinum-officium/tree/psnum/web/www/horas/Latin/psalms1.

I know that the references to psalms are included in many data files. But the number is not that huge. Currently, references to parts of psalms (which are the only problematic ones) are in 43 files (`grep -rl '[0-9]\+([0-9]\+-[0-9]\+)' horas`). Such references concerns 48 psalms. The changes can be performed incrementally and programmatically, psalm by psalm, along with simultaneous changes of the references for all languages.

Are there any other possible problems that may arise from fixing the numbering of verses?

**Comment by rl1234567890** [comment]
What about the translations? Shouldn't they follow the same numbering as the Latin? At least for aesthetic reasons.

**Comment by efati** [comment]
Of course they should! My example previewed only the concept taking Latin version as an example. I do plan to consistently update all languages.

**Comment by DLakomy** [comment]
On a sidenote: when I was arranging Polish translations of Psalms I adjusted the numbering to the Latin version, so it should be rather ok at the moment.

**Comment by efati** [comment]
OK. I'll write some scripts to check/update the numbering so every remaining inconsistencies should be revealed.

**Comment by efati** [comment]
It looks like that it is possible to semi-automatically update all translations except German, which is unusually structured (e.g. https://github.com/DivinumOfficium/divinum-officium/blob/master/web/www/horas/Deutsch/psalms1/Psalm4.txt, 5, 8, 13, 14 and others). I would skip these problematic psalms, and fix the others, which won't break the service.

**Comment by igneus** [comment]
Warning: in some cases a "breviary verse" spans over two "Bible numbering verses" or vice versa. I'm not 100% sure about the Gallican psalter (I could check and find examples if it was important), but there are quite a lot of such cases in Psalterium Pianum (ref #817 )

**Comment by igneus** [comment]
Warning 2: The verse numbering is used to split longer psalms in divisions. It must be made sure that even after an eventual renumbering the psalms are being divided correctly, placing each division after the same verse as in the breviary.
